mobile/src/components/scan/CaptureMarker.jsx

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
```

Comment:
since this component encompasses many things in the scanne page, would it make sense to change the file name to make it more descriptive? e.g., ScannerBackground, etc

Reply:
Changed with c33f245c2d996f1a3acb7f294635d88577349a2e